pages/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM dev-standard
+
+ENV _PAGES_VERSION 0.2.1
+ENV _JEKYLL_VERSION 3.1.2
+RUN bash -l -c "npm install -g 18f-pages-server@$_PAGES_VERSION && \
+    gem install jekyll -v $_JEKYLL_VERSION"
+
+RUN groupadd -r pages && useradd -r -m -g pages pages && \
+    mkdir -p $APP_SYS_ROOT/pages/repos $APP_SYS_ROOT/pages/sites && \
+    chown pages:pages $APP_SYS_ROOT/pages/repos $APP_SYS_ROOT/pages/sites
```

Comment:
For the images that need to `git clone` things (`pages`, and eventually `hub` and `team-api`), we need to create writable volumes, yes. In #5, you can see where `$APP_SYS_ROOT/pages/sites` gets broken out into its own data container, so that `pages` can write to it and `nginx-18f` can read from it. Even so, `pages` still creates its own `$APP_SYS_ROOT/pages/repos` volume to check out the GitHub repos for each Pages site, naturally (but no other container needs to access it).